pkgs/top-level/all-packages.nix

```diff
@@ -24695,6 +24695,20 @@ in
     gtk3 = if stdenv.isDarwin then gtk3-x11 else gtk3;
   });
 
+  vim_darwin = (vim_configurable.override {
+    config = {
+      vim = {
+        gui = "none";
+        darwin = true;
+      };
+    };
+  }).overrideAttrs (oldAttrs: rec {
+    pname = "vim_darwin";
+    meta = {
+      platforms = [ "x86_64-darwin" ];
```

Comment:
Shouldn't this be `platforms = platforms.darwin;`? to cover `aarch64-darwin` in the future